contents.json

```diff
@@ -17175,6 +17175,31 @@
       ],
       "date_added": "Dec 6 2018",
       "suggested_by": "@HeiKki"
+     },
+     {
+      "title": "ArtShredder",
+      "category-ids": [
+        "photo",
+        "content"
+      ],
+      "description": "Banksy Shredder for iOS",
+      "source": "https://github.com/marty-suzuki/ArtShredder",
+      "homepage": "https://marty-suzuki.github.io/ArtShredder/",
+      "itunes": "https://itunes.apple.com/app/id1439126672",
+      "license": "mit",
+      "lang": "zho",
+      "stars": 0,
+      "tags": [
+        "swift",
+        "arkit",
+        "banksy"
+      ],
+      "screenshots": [
```

Comment:
@marty-suzuki I don't see a difference between links. However, I can open the first link on the browser but the second link suggest me to download the gif. Do you have any thought why it is so?